compiler/rustc_codegen_ssa/src/back/link.rs

```diff
@@ -1087,16 +1087,17 @@ fn link_natively(
     let strip = sess.opts.cg.strip;
 
     if sess.target.is_like_osx {
+        let stripcmd = "/usr/bin/strip";
```

Comment:
Discussion: I don't know how to feel about this being hard-coded...

Reply:
The reliance on environment variables may be even less reliable and could cause the problems described above. I think the handling of OS X should be consistent with the other target.platforms in this file, so the hardcoded approach is necessary. A better approach might be to read the stripcmd as a rustflag...

Reply:
I think it's okay to hardcode in this case, not ideal that it's necessary, but we do it for other targets because oftentimes there's a system binary that is guaranteed to work and one in the path that may not.